backend/controllers/healthCheck.controller.js

```diff
@@ -0,0 +1,7 @@
+const HealthCheckController = {};
+
+HealthCheckController.isAlive = (_, res) => {
+  res.status(200).send("I'm Alive!");
+}
+
+module.exports = HealthCheckController;
```

Comment:
please, add an empty line at the end of the file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline